conf/igenomes.config

```diff
@@ -7,154 +7,391 @@
-       gtf         = "${params.igenomes_base}/Saccharomyces_cerevisiae/Ensembl/R64-1-1/Annotation/Genes/genes.gtf"
-       macs_gsize  = "1.2e7"
-     }
-     'EF2' {
-       fasta       = "${params.igenomes_base}/Schizosaccharomyces_pombe/Ensembl/EF2/Sequence/WholeGenomeFasta/genome.fa"
-       bwa         = "${params.igenomes_base}/Schizosaccharomyces_pombe/Ensembl/EF2/Sequence/BWAIndex/genome.fa"
-       gtf         = "${params.igenomes_base}/Schizosaccharomyces_pombe/Ensembl/EF2/Annotation/Genes/genes.gtf"
-       macs_gsize  = "1.21e7"
-     }
-     'Sbi1' {
-       fasta       = "${params.igenomes_base}/Sorghum_bicolor/Ensembl/Sbi1/Sequence/WholeGenomeFasta/genome.fa"
-       bwa         = "${params.igenomes_base}/Sorghum_bicolor/Ensembl/Sbi1/Sequence/BWAIndex/genome.fa"
-       gtf         = "${params.igenomes_base}/Sorghum_bicolor/Ensembl/Sbi1/Annotation/Genes/genes.gtf"
-     }
-     'Sscrofa10.2' {
-       fasta       = "${params.igenomes_base}/Sus_scrofa/Ensembl/Sscrofa10.2/Sequence/WholeGenomeFasta/genome.fa"
-       bwa         = "${params.igenomes_base}/Sus_scrofa/Ensembl/Sscrofa10.2/Sequence/BWAIndex/genome.fa"
-       gtf         = "${params.igenomes_base}/Sus_scrofa/Ensembl/Sscrofa10.2/Annotation/Genes/genes.gtf"
-     }
-     'AGPv3' {
-       fasta       = "${params.igenomes_base}/Zea_mays/Ensembl/AGPv3/Sequence/WholeGenomeFasta/genome.fa"
-       bwa         = "${params.igenomes_base}/Zea_mays/Ensembl/AGPv3/Sequence/BWAIndex/genome.fa"
-       gtf         = "${params.igenomes_base}/Zea_mays/Ensembl/AGPv3/Annotation/Genes/genes.gtf"
-     }
-     'hg38' {
-       fasta       = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Sequence/WholeGenomeFasta/genome.fa"
-       bwa         = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Sequence/BWAIndex/genome.fa"
-       gtf         = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Annotation/Genes/genes.gtf"
-       blacklist   = "${baseDir}/assets/blacklists/hg38-blacklist.bed"
-       macs_gsize  = "2.7e9"
-     }
-     'hg19' {
-       fasta       = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Sequence/WholeGenomeFasta/genome.fa"
-       bwa         = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Sequence/BWAIndex/genome.fa"
-       gtf         = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Annotation/Genes/genes.gtf"
-       blacklist   = "${baseDir}/assets/blacklists/hg19-blacklist.bed"
-       macs_gsize  = "2.7e9"
-     }
-     'mm10' {
-       fasta       = "${params.igenomes_base}/Mus_musculus/UCSC/mm10/Sequence/WholeGenomeFasta/genome.fa"
-       bwa         = "${params.igenomes_base}/Mus_musculus/UCSC/mm10/Sequence/BWAIndex/genome.fa"
-       gtf         = "${params.igenomes_base}/Mus_musculus/UCSC/mm10/Annotation/Genes/genes.gtf"
-       blacklist   = "${baseDir}/assets/blacklists/mm10-blacklist.bed"
-       macs_gsize  = "1.87e9"
-     }
+params {
+  // illumina iGenomes reference file paths
+  genomes {
+    'GRCh37' {
```

Comment:
Only one concern with such a big list, should we sort it?

Reply:
@MaxUlysse ❤️ sorting 📶 😆

Reply:
Im not sure about sorting the genomes in `igenomes.config`. Since we now have Ensembl and UCSC genomes in that file it would make sense to at least keep those separate. Also, most people are likely to use human and mouse assemblies which are at the top of that file. Just makes it easier to find those straight away. I think you also have to bear in mind that it will be very easy for the sorting to get messed up via template merges to this file. Not sure its worth the effort to be honest but happy to listen to opinions.